### ISSUE TITLE
App is crashing report

### ISSUE THREAD

**Issue by tugcekeser** [urbanairship/android-library#20]
Hi, 
I implemented urban-airship in the my app and my app is now in play store. I am getting some weird crash reports from New Relic. You can see at the attached files. Could you help me about this issue?

<img width="1145" alt="screen shot 2017-08-14 at 4 07 25 am" src="https://user-images.githubusercontent.com/17666583/29269419-35537ae2-80a6-11e7-88c6-7f1275f7811c.png">
<img width="1166" alt="screen shot 2017-08-14 at 4 07 41 am" src="https://user-images.githubusercontent.com/17666583/29269438-491d2172-80a6-11e7-96f5-ee4ed4b90837.png">

**Comment by rlepinski** [comment]
Could you post your SDK version? Are you using ADM or just GCM/FCM?

**Comment by tugcekeser** [comment]
I am only using GCM/FCM and my app compileSdkVersion is 26.

**Comment by tugcekeser** [comment]
In addition, all crashes are coming from the same environment.  We don't have any developer who will run the code on Emulator in Oregon Area or we are not serving in that area again. Could it be related to your environment?

**Comment by rlepinski** [comment]
@tugcekeser If you are not using ADM then its most likely issue #18 that was fixed in 8.8.0.
We do not have the ability to trigger that part of the code remotely without going through ADM, so its just a
coincidence that its only coming from Oregon.


AdmPushReceiver is installed with our SDK, but normally is not triggered on
standard Android devices (only Amazon Fire devices). Most likely this is a malicious app
installed on the users device that is trying to inject data into your app through
that receiver. Our SDK will do additional permissions and device checks before
processing that intent, but we were missing a null bundle check which is causing a NPE.

You have 2 ways of fixing this. Either update to 8.8.0 or remove the receiver by
using the merge manifest utils:

```
<receiver xmlns:tools="http://schemas.android.com/tools"
                  android:name="com.urbanairship.push.AdmPushReceiver"
                  tools:node="remove" />

```

If you are updating the SDK, we will be releasing 8.8.1 later to address #19.

**Comment by tugcekeser** [comment]
@rlepinski Thanks for your help! I will definitely make these changes on my code and let you know if I receive same crashes  again.
